backend/api/v1/endpoints/auth.py

```
@@ -3,16 +3,14 @@
 from database.session import SessionLocal
 import models.user as models
 from schemas.auth import Token
-from schemas.user import UserCreate
+from schemas.user import UserCreate, ProfileCompletionRequest
```

Ruff (0.8.2)

`schemas.user.ProfileCompletionRequest` imported but unused

Remove unused import: `schemas.user.ProfileCompletionRequest`

(F401)